### DISCUSSION TITLE
paying for work to get done in a specific region

### DISCUSSION THREAD

**Discussion by alibama** [osmandapp/OsmAnd#11022] [General]
hi - do you have any mechanisms in place for paying for mapping to get done in a certain region?  if we get a budget for $$s and want to set aside bitcoin or cardano or whatever for work to get done... is that anything your ecosystem supports?

**Comment by vshcherb** [comment]
Nope, OsmAnd doesn't build a community and it only supports. I'm afraid it's better and easier to contact Local OSM Chapter

**Reply by alibama** [reply]
yeah - we're trying to create a local OSM chapter by paying a living wage in our target area of rural guatemala... blockchain payments seem like a good way to encourage that kind of work, but it's not an obvious process for making it happen

anyhow - thanks for your feedback & have a great day!

**Reply by vshcherb** [reply]
well I think OsmAnd Live  could stimulate a bit local community (everybody could see stats / small incentives), but it couldn't create it. That's I think what you're interested in but that's a long way to go.

**Reply by alibama** [reply]
perfect - that's all i needed to know.  i'm working on a couple of grant proposals and need to get a sense of the space.  i'm very new here, but generally looking at chainlink onboarding of data & smart contract processing.... that said there's a whole world of aspects to these projects that make them interesting... and so much to learn... ugh...
